src/pages/AddMembersPage/NewMemberList.tsx

```diff
@@ -55,6 +57,7 @@ const NewMemberList = ({
         .object({
           address: zEthAddress.or(z.literal('')),
           name: zUsername.or(z.literal('')),
+          entrance: z.string().optional(),
```

Comment:
Out of curiosity, why make this optional? Don't we want to record this everywhere? Zod also know about typescript enums if you want to get extra fancy.

Reply:
How about using this commit: https://github.com/coordinape/coordinape/commit/7e1281cc739b9e144dd25b509beffb143c929fa6

- We make the field mandatory.
- We create a hidden form field to hold the data

I tried it and validaiton still passes and the values are correct as well.
@topocount @sshmm thoughts?

Reply:
What is the benefit of doing that? this will only be used in two cases CSV or manual address entry and the user will not have access to modify that so it doesn't need validation or to be mandatory

Reply:
@crabsinger  I cherry picked your commit and tested it. it had an issue if you tried to manually add after csv because the field doesn't have a default value to reset to. I fixed it in [5273807](https://github.com/coordinape/coordinape/pull/1504/commits/52738076b0a85c6e91a5b3d45448b76e8bb089db)

Reply:
@sshmm I think the benefit is that its more apparent that this value is part of the form, and allows us to enforce the non-null zod validation, and potentially easier to reason about in the future, but could agree with it being fine to do it another way. Curious @topocount thoughts on if he thinks this is better or not.